README.md

```diff
@@ -1,19 +1,61 @@
-# Experimental Design Library
+experimental-design-lib
+-----------------------------------
+# <p align=center>Experimental Design Library</p>
 
-[![Build Status](https://travis-ci.org/qbicsoftware/experimental-design-lib.svg?branch=development)](https://travis-ci.org/qbicsoftware/experimental-design-lib)[![Code Coverage]( https://codecov.io/gh/qbicsoftware/experimental-design-lib/branch/development/graph/badge.svg)](https://codecov.io/gh/qbicsoftware/experimental-design-lib)
+<p align="center"><i>Library for parsing of different experimental design formats and mapping to openbis properties </i></p>
 
-Parsers, writers for experimental design formats etc.
+<div align="center">
 
-## Author
 
-Created by Andreas Friedrich (andreas.friedrich@qbic.uni-tuebingen.de).
 
-## Description
+[![Build Maven Package](https://github.com/qbicsoftware/experimental-design-lib/actions/workflows/build_package.yml/badge.svg)](https://github.com/qbicsoftware/experimental-design-lib/actions/workflows/build_package.yml)
+[![Run Maven Tests](https://github.com/qbicsoftware/experimental-design-lib/actions/workflows/run_tests.yml/badge.svg)](https://github.com/qbicsoftware/experimental-design-lib/actions/workflows/run_tests.yml)
+[![CodeQL](https://github.com/qbicsoftware/experimental-design-lib/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/qbicsoftware/experimental-design-lib/actions/workflows/codeql-analysis.yml)
+[![release](https://img.shields.io/github/v/release/qbicsoftware/experimental-design-lib?include_prereleases)](https://github.com/qbicsoftware/experimental-design-lib/releases)
 
-## How to Install
+[![license](https://img.shields.io/github/license/qbicsoftware/experimental-design-lib)](https://github.com/qbicsoftware/experimental-design-lib/blob/main/LICENSE)
+![language](https://img.shields.io/badge/language-java-blue.svg)
 
-## License
+</div>
 
-This project is licensed under an MIT License:
+## How to run
+
+Create a runable version of this code with maven and java 8:
+
+```
+mvn clean package
+```
+
+The JAR file will be created in the ``/target`` folder
+
+## How to use
+
+This library is not hosted on maven central. To use it, you have to include our artifact repository to your pom.
+
+```xml
+<repositories>
+    <repository>
+        <releases>
+            <enabled>true</enabled>
+            <updatePolicy>always</updatePolicy>
+            <checksumPolicy>fail</checksumPolicy>
+        </releases>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+        <id>nexus-releases</id>
+        <name>QBiC Releases</name>
+        <url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-releases</url>
+    </repository>
+</repositories>
+```
+
+Then include this library as an artifact.
+```xml
+<dependency>
+    <groupId>life.qbic</groupId>
+    <artifactId>experimental-design-lib</artifactId>
+    <version>[version-number]</version>
+</dependency>
+```
```

Comment:
Very nice readme. I am missing a description on the data model as well as what exactly this library aims to provide.
Maybe an example how to use this library to parse experimental properties to openbis